### PR TITLE
Explore

### PR DESCRIPTION
**Pull request by tanveertkd**
Features added:  
- Explore
- User feed sorting
- Users can now add profile pictures
- Single page profile for users

### CODE REVIEW

File: src/components/EditProfileModal/EditProfileModal.jsx [hunk 1]
```diff
@@ -14,24 +18,72 @@ const EditProfileModal = ({ setModal, currentUser }) => {
...
                 className="login-input-form w-10/12"
                 onSubmit={(event) => editHandler(event, userInput)}
             >
+                <div className="form-input my-4 flex flex-col items-center">
+                    <label className="avatar relative w-max cursor-pointer mx-auto my-2">
+                        <img
+                            src={'https://i.pravatar.cc/200'}
+                            alt="profile-avatar"
+                            className="rounded-full w-fit"
+                        />
```
Comment (himadri2110) [inline src/components/EditProfileModal/EditProfileModal.jsx:72]: Suggestion: You can show the updated profile image on the edit profile modal by using `URL.createObjectURL`

**Review by himadri2110 — APPROVED** [review]
Check if the input is empty for both postInput and commentInput.
The sorting and editing profile image functionalities work fine.
Good to go.

File: src/components/EditProfileModal/EditProfileModal.jsx (hunk 1 above)
Comment (tanveertkd) [reply src/components/EditProfileModal/EditProfileModal.jsx:72]: Made the changes. Thanks!

**Review by msdeshmukh009 — APPROVED** [review]
The edit profile picture is working as expected.
Sorting is also working as expected. 
Good to go.

**Pull request merged**